### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by Legendarius-K**
Added a footer. waddaya think?

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -14,10 +14,10 @@
 <body>
   <p>Hello Earth!</p>
 
-  <footer>
-    <p>Fot!</p>
-    <p>Stortå</p>
-    <p>Lilltå</p>
+  <footer class="green">
+    <p>Esta es un footer!</p>
+    <p>Questo é un pié di pagina</p>
```
Comment (indiano881) [inline src/index.html]: piede inte pie mamma mia!!!

**Review by indiano881 — COMMENTED** [review]
piede

**Review by indiano881 — CHANGES_REQUESTED** [review]
bytta piede

File: src/index.html (hunk 1 above)
Comment (Legendarius-K) [reply src/index.html]: Ok then

**Pull request merged**